### DISCUSSION TITLE
How to completely remove/uninstall VPNHood from Windows (installed via Powershell commands)?

### DISCUSSION THREAD

**Discussion by geekyayush** [vpnhood/VpnHood#559] [General]
How to completely remove/uninstall VPNHood from Windows (installed via Powershell commands)?

**Comment by trudyhood** [comment]
Hi, Thank you for asking,
I add this for you:
https://github.com/vpnhood/VpnHood/wiki/How-to-uninstall-VpnHood-Server-from-Windows

Let me know if it there is any issue on it
